main.py

```diff
@@ -0,0 +1,18 @@
+def tokenize(text: str) -> list[str] | None:
```

Comment:
Please delete this file, all work is supposed to be located in the designated folder.